tedana/info.py

```diff
@@ -23,7 +23,6 @@
 __packagename__ = 'tedana'
 __description__ = ('TE-Dependent Analysis (tedana) of multi-echo functional '
                    'magnetic resonance imaging (fMRI) data.')
-__longdesc__ = readme_path.open().read()
```

Comment:
Since this is being dropped, you can drop the line that defines `readme_path` in this file, along with the `Path` import.

Reply:
Nice catch!